index.html

```diff
@@ -6,11 +6,91 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <div class="container">
+
+        <div class="icon_img box">
+            <img src="https://www.bing.com/th?id=OIP.5wHPVNq-5T5Adqavx62ouwAAAA&w=150&h=150&c=8&rs=1&qlt=90&o=6&pid=3.1&rm=2"
+                alt="icon" width="100px">
+        </div>
+
+        <div class="bor">
+            <ul class="bor_li box">
+                <li class="">Home</li>
+                <li class="">About</li>
+                <li class="">Blog</li>
+                <li class="">Contact</li>
+            </ul>
+        </div>
+
+        <div class="p_nav box">
+            <h3>The best cakes in town<br>delivered to your door</h3>
+        </div>
+
+        
+
+        <div class="wel_tex box">Welcom
+
+        </div>
+
+        <article class="artical box">Our new range of postable cakes
+            has arrived! You have been asking for a long time for
+            Cutter & Squidge cakes to be sent outside of London,
+            so we have been working hard developing this
+            collection of cakes that can be sent anywhere in
+            Manchester UK!
+        </article>
+
+
+
+
+
+
+
+
+
+        <div class="main_img box">
+            <img src="https://th.bing.com/th/id/OIP.hT1QYACXEUqs6uTy8DyC6AHaE7?w=247&h=180&c=7&r=0&o=5&pid=1.7"
+                alt="main cakes" height="100%" width="100%" />
+        </div>
+
+        <div class="fimg_pic box">
+            <img src="https://th.bing.com/th/id/OIP.ylWiJppALlJ_lrxBgSJMWwHaJ4?w=186&h=248&c=7&r=0&o=5&pid=1.7"
+                alt="img1-1" height="100%" width="100%">
```

Comment:
Remember that alt text should provide a textual representation of the meaning of the image. So in this case it would describe the cake in the photo.